packages/@webex/plugin-meetings/src/roap/request.ts

```diff
@@ -49,23 +66,24 @@ export default class RoapRequest extends StatelessWebexPlugin {
       const deviceUrl = this.webex.internal.device.url;
       let url = '';
 
+      const reachabilityData = this.attachReachabilityData({
```

Comment:
BTW, it looks like joinMeetingWithRoap() is not used anywhere and it's removed on beta, so not sure if you need to do this or just delete joinMeetingWithRoap()

Reply:
removed (PR based off beta now)